packages/design-system/src/styles/settings/variables/_build.scss

```diff
@@ -1,6 +1,6 @@
 // Paths
-$font-path: './fonts' !default;
-$image-path: './images' !default;
+$font-path: '../fonts' !default;
```

Comment:
Why is this change being made?

Reply:
Notice how if you view github pages, the flag image is not found. This fixes the issue of not locating the fonts & images when we view the `.html` file

Reply:
By setting the default to reference `../` folder, accesses to the fonts & images files works.